### PR TITLE
Consume cds-services 3.5.0

### PR DESCRIPTION
**Pull request by agoerler**

### CODE REVIEW

File: srv/src/main/java/my/bookshop/handlers/AdminServiceHandler.java [hunk 1]
```diff
@@ -94,14 +94,6 @@ public void beforeCreateOrder(Stream<Orders> orders, EventContext context) {
 				order.getItems().forEach(orderItem -> {
 					// validation of the Order creation request
 					Integer quantity = orderItem.getQuantity();
-					if (quantity == null || quantity <= 0) {
-						// errors with localized messages from property files
-						// exceptions abort the request and set an error http status code
-						// messages in contrast allow to collect multiple errors
-						messages.error(MessageKeys.QUANTITY_REQUIRE_MINIMUM)
```
Comment (beckermarc) [inline srv/src/main/java/my/bookshop/handlers/AdminServiceHandler.java:101]: The message removal seems to be missing from this PR.

File: srv/admin-service.cds [hunk 2]
```diff
@@ -32,6 +32,12 @@ annotate AdminService.Books with @cds.search : {
   title
 };
 
+annotate AdminService.OrderItems with  {
+  @mandatory
+  @assert.range: [(0), _]
+  quantity
+};
```
Comment (beckermarc) [inline srv/admin-service.cds:39]: Should we do this on db-level already? Most other validations like these seem to be defined there.

**Review by beckermarc — DISMISSED** [review]

File: srv/src/main/java/my/bookshop/handlers/AdminServiceHandler.java (hunk 1 above)
Comment (agoerler) [reply srv/src/main/java/my/bookshop/handlers/AdminServiceHandler.java:101]: The message is still used [here](https://github.com/SAP-samples/cloud-cap-samples-java/pull/402/files?diff=unified&w=0#diff-af608d1373e020896c7b3dfc5904d6e45e8e69219e1efa995396a5f5b060bb25L207). I think this is the warning, which is displayed on the UI. Can we get rid of this as well?

Comment (beckermarc) [reply srv/src/main/java/my/bookshop/handlers/AdminServiceHandler.java:101]: Ah good catch, I guess that can't be easily removed, if we want to keep the warning..

**Comment by agoerler** [comment]
@mofterdinger - please feel invited to review this PR ;-)

**Review by mofterdinger — APPROVED** [review]

**Pull request merged**